### ISSUE TITLE
Bug: Voicelines are being counted for player reports 

### ISSUE THREAD

**Issue by benndur** [ValveSoftware/Dota2-Gameplay#13394]
### Description

From a reddit thread:

https://www.reddit.com/r/DotA2/comments/17q237w/petition_to_publish_the_mechanism_of_the/k89fvj8/?context=3

"I've only been playing party unranked with my buddies since end of TI. Our stack is ( immortal(me pos 3), divine (pos 2), ancient(pos 1), ancient (pos 4), archon(pos 5).

1) We aren't all chatting with enemy. Literally don't have the need to.

2) We are in discord call together.

3) we have been using talent voicelines ( that we PAID for)

4) None of us are smurfs.

My behavior score dropped by 4k from being 11k, theirs by 2k to 6k.

Only thing makes sense is we are being reported for using voicelines.

Which is utterly ridiculous.

So yes something is out of whack."

It makes no sense that players can use communication reports on others for voice lines. 
It seems this is indeed the case right now, as even players under 3k behaviour score, who can't even communicate, are still getting communication reports.

This is likely a related to why there are so many people complaining about behaviour score drops right now.

**Comment by Waxz200** [comment]
I don't think its about voice lines. People are just abusing reports, and they'll drop your behavior score no matter if they are true or false right now. You don't even need to use chat at all, to get reported for commu abuse.

**Comment by benndur** [comment]
> I don't think its about voice lines. People are just abusing reports, and they'll drop your behavior score no matter if they are true or false right now. You don't even need to use chat at all, to get reported for commu abuse.

The user specifically said they only play 5 stacks and don't communicate with the enemy. They have no reason to use in-game communication since they are in discord with each other, and they obviously aren't reporting each other so by process of elimination that leaves the enemy abusing communication reports. 

"You don't even need to use chat at all, to get reported for commu abuse."

You just contradicted yourself. Why do you can get reported without using chat? That's the whole point. Voice lines/Chat wheel are likely making players eligible for reports.

**Comment by Waxz200** [comment]
I dont know dude. But my behavior is now 5k, dropped by 2k  by only playing turbo. My score keeps going down, i cant even chat or talk in the game now. But i keep getting alot of reports for Communication abuse, i dont even use voice lines.

**Comment by eric1294** [comment]
No one wants to hear your retarded talent voice lines. Keep your mouth shut and you won't get reported.

**Comment by eric1294** [comment]
Yes it should if it's annoying the hell out of other players. So in a nutshell you deserved the reports and the mutes.

**Comment by arturkrugol** [comment]
There is reallly actual thing in the topic.

**Comment by benndur** [comment]
> No one wants to hear your obnoxious talent voice lines. Keep your mouth shut and you won't get reported.

Sounds like you need to get better at the game buddy! If you're this mad that your enemies keep owning you and using voicelines on you, you should try getting better at the game! Being salty on github is not going to help you :)

**Comment by Waxz200** [comment]
Yes voice lines were made so people could use them lmao :D

**Comment by eric1294** [comment]
> > No one wants to hear your obnoxious talent voice lines. Keep your mouth shut and you won't get reported.
> 
> Sounds like you need to get better at the game buddy! If you're this mad that your enemies keep owning you and using voicelines on you, you should try getting better at the game! Being salty on github is not going to help you :)

Sounds like **you** need to learn to shut your STD infested mouth you HIV positive dickhead.

**Comment by rmk221** [comment]
Probably voiceline and ping related. I don't even mean pinging team items, even basic map pings might be reportable. It plummeted after using ephey voiceline a lot, then just keeps going down even though I don't use lines anymore.

**Comment by gitguddota2** [comment]
> Yes it should if it's annoying the hell out of other players. So in a nutshell you deserved the reports and the mutes.

Hey, there is a mute feature implemented for a reason. If you don't want to see it, mute them. Players have specifically paid a subscription fee to DotaPlus or bought talent voicelines and it is absolutely insane you are suggesting players should be punished for using in-game features.

That's like reporting the enemy for using a TP scroll instead of walking everywhere XD

Part of the behavior issue is players like you reporting players who are not griefing, feeding, AFK or intentionally ruining games. Just getting on your nerves by playing a funny voiceline :)

**Comment by benndur** [comment]
> > Yes it should if it's annoying the hell out of other players. So in a nutshell you deserved the reports and the mutes.
> 
> Hey, there is a mute feature implemented for a reason. If you don't want to see it, mute them. Players have specifically paid a subscription fee to DotaPlus or bought talent voicelines and it is absolutely insane you are suggesting players should be punished for using in-game features.
> 
> That's like reporting the enemy for using a TP scroll instead of walking everywhere XD
> 
> Part of the behavior issue is players like you reporting players who are not griefing, feeding, AFK or intentionally ruining games. Just getting on your nerves by playing a funny voiceline :)

Don't waste your time trying to use logic on trolls, it's simply an exercise in futility. 

> > > No one wants to hear your obnoxious talent voice lines. Keep your mouth shut and you won't get reported.
> > 
> > 
> > Sounds like you need to get better at the game buddy! If you're this mad that your enemies keep owning you and using voicelines on you, you should try getting better at the game! Being salty on github is not going to help you :)
> 
> Sounds like **you** need to go be a dick somewhere else. Not being salty here you walking STD.

Have you tried getting better at the game buddy? Maybe that's why you're so upset? Because youre garbage at 10 year old game? :) Sorry friend!

**Comment by TRANDAFIR77** [comment]
I got muted 2 years ago or something like that, by using only the voiceline from PL "such simple minds". I could still use it even though I was chat restricted for 48h. I got further reported for communicaiton abuse and went to 120h and tanked around 3k behaviour. When I sent them a ticket, as I stated that I paid for dota+, a premium service they offer or w/e, they said that they can't do anything. And I asked, what do you mean you can't do anything if I pay for dota+ to use the voice lines, yet I get muted and I cannot enjoy the game? I had 3 months ban after that. I took a break, was absolute impossible to play. went from 10k behaviour to around 6,2k or something like that back then. They will not fix it, it is an old issue and I am not the only one who experienced it.
